tests/runtest.cmd

```diff
@@ -57,6 +57,10 @@ if /i "%1" == "TestEnv"             (set __TestEnv=%2&shift&shift&goto Arg_Loop)
 if /i "%1" == "sequential"          (set __Sequential=1&shift&goto Arg_Loop)
 if /i "%1" == "crossgen"            (set __DoCrossgen=1&shift&goto Arg_Loop)
 if /i "%1" == "longgctests"         (set __LongGCTests=1&shift&goto Arg_Loop)
+if /i "%1" == "jitstree"            (set COMPlus_JitStress=%2&shift&shift&goto Arg_Loop)
+if /i "%1" == "jitstreeregs"        (set COMPlus_JitStressRegs=%2&shift&shift&goto Arg_Loop)
+if /i "%1" == "jitminopts           (set COMPlus_JITMinOpts=1&shift&shift&goto Arg_Loop)
+if /i "%1" == "jitforcerelocs"      (set COMPlus_ForceRelocs=1&shift&shift&goto Arg_Loop)
```

Comment:
These don't seem to match up with the arg names in the usage section below ('jitstree' vs 'jitstress', 'jitstreeregs' vs 'jitstressregs'